assets/manifest_template.toml

```diff
@@ -16,26 +16,25 @@ _FLOX_INIT_INSTALL
 [vars]
 # message = "Howdy"
 
-# Scripts defined in the `[profile]` section are sourced by your shell
-# immediately after entering the environment and provide a way to add shell
-# aliases to the environment or dynamically compute environment variables.  The
-# `profile.common` script is sourced by all shells, so you must write it to be
-# compatible with all shells. The `profile.bash` and `profile.zsh` scripts will
-# only be sourced by the corresponding shell after sourcing the `profile.common`
-# script.
-[profile]
-_FLOX_INIT_PROFILE
-
-# The script defined in `hook.on-activate` is run in a non-interactive Bash
-# sub-shell after sourcing the profile scripts. This script inherits all the
-# environment variables set in the `[vars]` and `[profile]` sections, but since
-# it runs in a sub-shell it can't modify them. The output and exit code of the
-# hook script is discarded. The utility of the `hook` script is that it runs in
-# a consistent shell environment so you don't have to worry about shell
-# compatibility during complicated initialization.
+# The `hook.on-activate` script is run by the *bash* shell immediately upon
+# activating an environment for the first time only. Subsequent activations,
```

Comment:
I agree brevity is always best but not everyone is going to read docs (who are we kidding, very few do) and this comment is likely our first and best opportunity to convey new concepts. Without dropping a teaser for "in-place" activations this comment is likely to be met by the user with just a "duh, that's obvious" sort of reaction, but as currently written it can potentially introduce the new concept of "in-place" activations to the user.

Regardless, the boilerplate text for the prototype manifest is really more of a decision for product and marketing. I reviewed the text with both Graham and Ross before logging the PR, but will be happy for them to suggest edits based on your comments. I'll go ahead and resolve your other comment on this topic and leave this one open until we've solicited their thoughts.

Reply:
I'll meet with Ross this afternoon and come up with a "today" version of the template so we can move forward, understanding that it will undoubtedly be a living bit of text as things evolve over time, and comparably far easier to change than the remainder of this PR. With that I'll go ahead and resolve this conversation and commit to post a followup of our meeting later this afternoon.